<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1131</width>
      <height>691</height>
     </rect>
    </property>
    <property name="title">
     <string>Scan Search</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>451</width>
       <height>641</height>
      </rect>
     </property>
     <property name="title">
      <string>Info from User</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pick a Keyword to Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>441</width>
        <height>271</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="imgText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>451</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Searching</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>681</width>
       <height>621</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <widget class="QTextBrowser" name="desc_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Side Effects</string>
      </attribute>
      <widget class="QTextBrowser" name="effect_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Uses</string>
      </attribute>
      <widget class="QTextBrowser" name="uses_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Dosage</string>
      </attribute>
      <widget class="QTextBrowser" name="dosage_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Substitutes</string>
      </attribute>
      <widget class="QTextBrowser" name="subs_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="keyw_databse">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="backtodash">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to Dashboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
